_pages/developers.md

```diff
@@ -7,41 +7,32 @@ permalink: /developers/
   <div class="container cntnr-wide px2 py5 clearfix">
     <img alt="" width="231" class="mt1 mx4 right md-show" src="{{ '/assets/img/playbook-landing.svg' | relative_url }}">
     <h1 class="mt0 mb1 white">
-      Built for easy integration
+      Easy integration and flexibility
     </h1><img alt="" class="mb3" src="{{ '/assets/img/hr-red-2.svg' | relative_url }}">
     <p class="mb3 overflow-hidden white fs-20p serif">
-      We speed up integration with open code repositories, standards-based support, and API-based interactions.
+      Standard protocols, reusable libraries and continuous open development speed up development and help you stay compliant.
```

Comment:
Also wondering whether developers use the word `feedback` -- I feel like I hear the word `review` (eg code review) more frequently.